pkg/userdata/ubuntu/testdata/dist-upgrade-on-boot.golden

```diff
@@ -306,12 +306,17 @@ write_files:
   content: |
     export PATH="/opt/bin:$PATH"
 
-- path: /etc/systemd/system/docker.service.d/10-storage.conf
+- path: /etc/docker/daemon.json
   permissions: "0644"
   content: |
-    [Service]
-    ExecStart=
-    ExecStart=/usr/bin/dockerd -H fd:// --storage-driver=overlay2
+    {
+      "storage-driver": "overlay2",
+      "log-driver": "json-file",
+      "log-opts": {
+        "max-size": "10m",
+        "max-file": "2"
```

Comment:
Is this a per-container setting?

Reply:
Unfortunately yes.

Reply:
Do we already limit the overall journald size?

Reply:
Yes, on every system we set:
https://github.com/kubermatic/machine-controller/pull/382/files#diff-8e277877f731647a2f08ec1c985fb9abR16